build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
We run the tests identically except `_k-test.shade` assumes `build.cmd` and `build.sh` ensure `$Path` contains the full CLR `default` runtime. If you want it a special case for DNX testing, add it for both runtime flavors. Main complication is that it'll be `{alias} -runtime clr` on Windows and `{alias} -runtime mono` everywhere else.

Also, I wasn't suggesting a change to the `default` alias. Add an alias for the just-built, just-installed runtime. Or add two -- one for DNX Core and one for Mono or CLR.